RELEASE.md

```diff
@@ -75,24 +75,29 @@ you can skip the dependency update or only update select dependencies. In such a
 case, you have to create an issue or pull request in the GitHub project for
 later follow-up.
 
-### Prepare your release
+### 1. Prepare your release
 
-For a patch release, work in the branch of the minor release you want to patch.
+Create a new release branch.
+
+For a patch release, work in the branch of the minor release you want to patch. So for 2.16.1, create a new branch off of 2.16.0.
 
 For a new major or minor release, create the corresponding release branch based on the master branch.
+** Note: ** this means that for a release candidate you should create the non-release candidate branch (release-2.16) and merge the RC versions (ex: cut-v2.16.0-rc.0) into it as you go.
 
-Bump the version in the `VERSION` file and update `CHANGELOG.md`. Do this in a proper PR pointing to the release branch as this gives others the opportunity to chime in on the release in general and on the addition to the changelog in particular.
+Bump the version in the `VERSION` file and update `CHANGELOG.md`. Do this in a proper PR pointing to the release branch as this gives others the opportunity to chime in on the release in general and on the addition to the changelog in particular. For a release candidate, append something like `-rc.0` to the version (with the corresponding changes to the tag name, the release name etc.).
 
 Note that `CHANGELOG.md` should only document changes relevant to users of Prometheus, including external API changes, performance improvements, and new features. Do not document changes of internal interfaces, code refactorings and clean-ups, changes to the build process, etc. People interested in these are asked to refer to the git history.
 
+For release candidates still update `CHANGELOG.md`, but when you cut the final release later, merge all the changes from the pre-releases into the one final update.
+
 Entries in the `CHANGELOG.md` are meant to be in this order:
 
 * `[CHANGE]`
 * `[FEATURE]`
 * `[ENHANCEMENT]`
 * `[BUGFIX]`
 
-### Draft the new release
+### 2. Draft the new release
 
 Tag the new release with a tag named `v<major>.<minor>.<patch>`, e.g. `v2.1.3`. Note the `v` prefix.
```

Comment:
As this was never updated for RCs, let's do it now. This could just refer to the content of the VERSION file, is it is also implemented in the lines below. Or it should mention again the possible suffix in case this is a RC.

Reply:
Not sure what you're saying should change here?

Reply:
```suggestion
Tag the new release with a tag named `v<major>.<minor>.<patch>`, e.g. `v2.1.3`. Note the `v` prefix. Pre-releases should get a suffic `-rc.x`, e.g. `v2.1.3-rc.0`.
```

Reply:
This just doesn't seem necessary with the explanation of the version scheme above + changing the `VERSION` file, and then the instructions to create the tag below. Should we change this section to just be `Tag the new release via the following commands`?

Reply:
Yes, that would be best.